js/main.js

```diff
@@ -1,13 +1,223 @@
+                label += _.sample(vocabulary.filthy) + ' ';
+            } else {
+                label += _.sample(vocabulary.sublime) + ' ';
+            }
+            label += _.sample(vocabulary.scum) + '.';
+            break;
+
+        case 2:
+            // eg, "bastards of the internet"
+            label += _.sample(vocabulary.bastards) + ' of the ';
+            label += _.sample(vocabulary.internet) + '.';
+            break;
+
+        case 3:
+            // eg, "sexy beasts of the apocalypse"
+            if (Math.random() < 0.5) {
+                label += _.sample(vocabulary.sexy) + ' ';
+            }
+            label += _.sample(vocabulary.beasts) + ' of ';
+            label += _.sample(vocabulary.ill_omen) + '.';
+            break;
+
+        case 4:
+            // eg, "internet scum horde"
+            label += _.sample(vocabulary.internet) + ' ';
+            label += singularize(_.sample(vocabulary.scum)) + ' ';
+            if (Math.random() < 0.5) {
+                label += _.sample(vocabulary.horde) + '.';
+            } else {
+                label += 'of ' + _.sample(vocabulary.sexiness) + '.';
+            }
+            break;
+
+        case 5:
+            // eg, "walking slabs of haunted pork"
+            label += _.sample(vocabulary.arbitrary_gnarbags) + '.';
+            break;
+
+        case 6:
+            // eg, "filthy sex polyps"
+            label += _.sample(vocabulary.filthy) + ' ';
+            label += _.sample(vocabulary.sex) + ' ';
+            label += _.sample(vocabulary.body_parts) + '.';
+            break;
+
+        default:
+            // eg, "you glowing sex animals of the internet"
+            label += 'you ' + _.sample(vocabulary.sublime) + ' ';
+            label += _.sample(vocabulary.sex) + ' ';
+            if (Math.random() < 0.5) {
+                label += _.sample(vocabulary.mammals);
+            } else {
+                label += 'of ' + _.sample(vocabulary.beasts);
+            }
+            if (Math.random() < 0.75) {
+                label += ' of the ' + _.sample(vocabulary.internet) + '.';
+            } else {
+                label += '.';
+            }
+    }
+    return label;
+};
+
+//
+// Good Morning Scum
+//
+// Produces a quote like: "Good Morning, Bastards of the Internet"
+//
+var goodMorningScum = function(vocabulary) {
+    var greeting = '';
+
+    // eg, Good Morning
+    greeting += _.sample(vocabulary.salutation) + ' ';
+
+    // eg, "bastards of the internet"
+    greeting += ellisLabel(vocabulary);
+
+    if (Math.random() < 0.1) {
+        // eg, "I hate you"
+        greeting += ' ' + _.sample(vocabulary.extra);
+    }
+
+    return greeting;
+};
+
+//
+// Attention Scum
+//
+// Produces a quote like "ATTENTION SCUM: I will destroy you"
+//
+var attentionScum = function(vocabulary) {
+    var greeting = '';
+
+    if (Math.random() < 0.5) {
+        // eg, "ATTENTION SCUM"
+        greeting += _.sample(vocabulary.attention) + ' ';
```

Comment:
Not needed in this case, because the source text is uppercase, but now that you mention it, I agree it's confusing. I'll go lowercase the source text and uppercase it here.